tutorials/README.md

```diff
@@ -9,18 +9,121 @@ Learn how to quickly quantize pre-trained models using MCT's post-training quant
 - [Post training quantization with Keras](notebooks/imx500_notebooks/keras/example_keras_mobilenetv2_for_imx500.ipynb)
 - [Post training quantization with PyTorch](notebooks/mct_features_notebooks/pytorch/example_pytorch_post_training_quantization.ipynb)
 
-## MCT Features
-This set of tutorials covers all the quantization tools provided by MCT. 
-The notebooks in this section demonstrate how to configure and run simple and advanced post-training quantization methods.
-This includes fine-tuning PTQ (Post-Training Quantization) configurations, exporting models,
-and exploring advanced compression techniques. 
+## MCT features
+This tutorial set introduces the various quantization tools offered by MCT. 
+The notebooks included here illustrate the setup and usage of both basic and advanced post-training quantization methods. 
+You'll learn how to refine PTQ (Post-Training Quantization) settings, export models, and explore advanced compression 
+techniques such as GPTQ (Gradient-Based Post-Training Quantization), Mixed precision quantization and more.
 These techniques are essential for further optimizing models and achieving superior performance in deployment scenarios.
-- [MCT Features notebooks](notebooks/mct_features_notebooks/README.md)
 
-## Quantization for Sony-IMX500 deployment
+### Keras Tutorials
 
-This section provides several guides on quantizing pre-trained models to meet specific constraints for deployment on the
-[Sony-IMX500](https://developer.sony.com/imx500/) processing platform. 
-We will cover various tasks and demonstrate the necessary steps to achieve efficient quantization for optimal
-deployment performance.
-- [MCT IMX500 notebooks](notebooks/imx500_notebooks/README.md)
+<details id="keras-ptq">
+  <summary>Post-Training Quantization (PTQ)</summary>
+  
+  | Tutorial                                                                                   | Included Features                                                                                   |
+  |--------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------|
+  | [Basic Post-Training Quantization (PTQ)](keras/example_keras_post-training_quantization.ipynb)          | &#x2705; PTQ                                                                                        |
+  | [MobileNetV2](../imx500_notebooks/keras/example_keras_mobilenetv2_for_imx500.ipynb)        | &#x2705; PTQ                                                                                        |
```

Comment:
there are several tutorials here that refer to IMX500 tutorials, they need to be removed